assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * ReminderBanner component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BannerNotification from '../../../../../components/notifications/BannerNotification';
+
+export default function ReminderBanner( { onCTAClick } ) {
+	return (
+		<BannerNotification
+			id="ga4-activation-banner"
+			title={ __(
+				'Set up Google Analytics 4 now to join the future of Analytics',
+				'google-site-kit'
+			) }
+			ctaLabel={ __( 'Set up now', 'google-site-kit' ) }
+			ctaLink={ onCTAClick ? '#' : null }
```

Comment:
We ready should avoid `href="#"` when possible… if we omit that will this render as a button instead of a link? That'd be ideal 🤔

Reply:
If we don't pass a `ctaLink` prop, neither a button nor link would show up. We can do some refactoring in the `BannerNotification` component where it would show the button if one of `ctaLink` or `onCTAClick` are present. But since it is a heavily used component, we decided not to make any changes to `BannerNotification` in #5550 which uses the same approach of passing "#".

Reply:
Noted! Alright, fair enough, that can be a separate issue. Thanks for the explanation 🙂